docs/pages/indexer.mdx

```diff
@@ -290,3 +290,82 @@ pnpm start:sqlite
 ```
 
 Note that in your queries you need to specify the correct chainId for the chain you are indexing.
+
+## Using the indexer with PostgreSQL
+
+To run the indexer with [PostgreSQL](https://www.postgresql.org/), follow these steps:
+
+1. Download and install [PostgreSQL](https://www.postgresql.org/download/).
+
+1. To use the default setting (local PostgreSQL, using the `postgres` database, indexing the local `anvil` chain), run:
+
+   ```sh copy
+   pnpm start:postgres:local
+   ```
+
+   Otherwise, set these environment variables:
+
+   | Variable     | Value                                                            |
+   | ------------ | ---------------------------------------------------------------- |
+   | DATABASE_URL | URL for the database, of the form `postgres://<host>/<database>` |
+   | CHAIN_ID     | ChainID for the chain you index                                  |
```

Comment:
we prob want to list the chain_id we currently support (and a link to that file with constants), along with instruction on how to bring custom rpc and chain_id in case it is not listed in the MUD constant file

Reply:
Done. I assume we use the same mechanism we do with sqlite, environment variables.